src/ResourceManager/MachineLearningCompute/ChangeLog.md

```diff
@@ -19,6 +19,13 @@
 -->
 ## Current Release
 
+## Version 0.3.0
```

Comment:
@shutchings please remove this `## Version 0.3.0` header and move the below contents to be under `## Current Release`

Reply:
Done.